components/drata/actions/upload-device-compliance-document/upload-device-compliance-document.mjs

```diff
@@ -0,0 +1,111 @@
+import drata from "../../drata.app.mjs";
+import { axios } from "@pipedream/platform";
+
+const docsLink = "https://developers.drata.com/docs/openapi/reference/operation/DevicePublicController_uploadDocumentForDevice/";
+
+export default {
+  key: "drata-upload-device-compliance-document",
+  name: "Upload Device Compliance Document",
+  description: `Upload a device compliance document. [See docs here.](${docsLink})`,
+  version: "0.0.1",
+  type: "action",
+  props: {
+    drata,
+    deviceId: {
```

Comment:
Can you implement async options from https://developers.drata.com/docs/openapi/reference/operation/DevicePublicController_getDeviceDocuments/ ?

Reply:
This endpoint is to list the documents for a specified `deviceId`